src/Domain/TagsTimeDependent.hpp

```diff
@@ -27,25 +27,32 @@ namespace domain {
 /// \ingroup ComputationalDomainGroup
 /// \brief %Tags for the domain.
 namespace Tags {
-/// The Inertial coordinates, Inertial grid velocity, and the Jacobian and
-/// inverse Jacobian from the Grid to the Inertial frames.
+/// The Inertial coordinates, the inverse Jacobian from the Grid to the Inertial
+/// frame, the Jacobian from the Grid to the Inertial frame, and the Inertial
+/// mesh velocity.
+///
+/// The type is a `boost::optional`, which, when is not valid, signals that the
+/// mesh is not moving. Thus,
+/// `static_cast<bool>(coordinates_velocity_and_jacobian)` can be used to check
+/// if the mesh is moving.
 template <size_t Dim>
-struct CoordinatesVelocityAndJacobians : db::SimpleTag {
+struct CoordinatesMeshVelocityAndJacobians : db::SimpleTag {
```

Comment:
[nitpicking] To reflect the quantities it holds this should probably be `CoordinatesJacobiansAndMeshVelocity`

Reply:
Meh, it'll be annoying to deal with more renames and this tag is never used by people directly anyway :man_shrugging: